app/views/collections/_collection.html.erb

```diff
@@ -1,7 +0,0 @@
-<% desc = collection['description_tesim'] %>
```

Comment:
Overriding the default partial here is why Collections rendered so differently in search results, with no item number and special formatting.